### ISSUE TITLE
phpcbf can only insert spaces for indent

### ISSUE THREAD

**Issue by fizzka** [squizlabs/PHP_CodeSniffer#467]
When i use convention with tab indents, I can't get a correct tab padding with `phpcbf`.

f.e.
```php
<?php
	function example() {}
```
^^^
here 1 tab before `function` keyword

maximum i can get is visual alignment with command:
```shell
phpcbf --standard=Squiz --sniffs=Squiz.WhiteSpace.ScopeClosingBrace --tab-width=4 --report=diff ex.php
```

output is:
```php
<?php

	function example() {
    }
```
^^^
here `function` is 1-tab indented, but `}` is 4-spaced

---
I suppose problem is in` ScopeClosingBraceSniff.php:112` (only space-indentation):
```php
... str_repeat(' ', $diff) ...
```

**Comment by aik099** [comment]
If you want tabs to be used, then you need custom standard for that, because all built-in standards use spaces for indentation.

To get this right you must have this in your `ruleset.xml`:
```xml

<arg name="tab-width" value="4" />
<rule ref="Generic.WhiteSpace.DisallowSpaceIndent"/>

<rule ref="Generic.WhiteSpace.ScopeIndent" phpcs-only="true">
	<properties>
		<property name="tabIndent" value="true"/>
	</properties>
</rule>
```

**Comment by aik099** [comment]
It works like this:
1. all fixes in sniffs are done using spaces
2. there 2 sniffs that then (if configured) replaced N spaces with 1 tab (the `Generic.WhiteSpace.DisallowSpaceIndent` and `Generic.WhiteSpace.ScopeIndent` ones)
3. how much spaces are in one tab is set via `tab-width` setting, which is madatory in this case

**Comment by fizzka** [comment]
thanks a lot!

seems to be working!
